easybuild/framework/easyblock.py

```diff
@@ -2108,6 +2127,9 @@ def _sanity_check_step(self, custom_paths=None, custom_commands=None, extension=
                     self.sanity_check_fail_msgs.append("no %s of %s in %s" % (typ, xs, self.installdir))
                     self.log.warning("Sanity check: %s" % self.sanity_check_fail_msgs[-1])
 
+                cand_paths = ' or '.join(["'%s'" % x for x in xs])
+                trace_msg("%s %s found: %s" % (typ, cand_paths, ('FAILED', 'OK')[found]))
```

Comment:
Don't you want to print here first a line what you're about to print?

Reply:
that's done higher up, output looks like:

```
    == sanity checking...
      >> file 'bin/h52gif' found: OK
      >> file 'bin/h5c++' found: OK
```